protecode/model.py

```diff
@@ -98,7 +103,9 @@ def historical(self):
     def cve(self):
         return self.raw.get('vuln').get('cve')
 
-    def cve_severity_str(self):
+    def cve_severity_str(self, cvss_version=CVSSVersion.V2):
+        if cvss_version == CVSSVersion.V3:
```

Comment:
- use `is` to check enum values
- use explicit `if`, `elif`, `else`, raise `NotImplementedError` in `else`